File: Source/server/rabbit-game/src/SharedModel/ClientIntentions/ReadyForReplayIntention.cs

namespace RabbitGameServer.SharedModel.ClientIntentions
{
	public class ReadyForReplayIntention : ClientIntention
	{
		public string roomId { get; set; }

		public ReadyForReplayIntention(string username, string roomId)
			: base(ClientIntentionType.ReadyForReplay, username)
		{
			this.roomId = roomId;
		}
	}
}

File: Source/client-model/src/main/java/model/intention/ReadyForReplayIntention.java
package model.intention;

import root.model.event.ClientIntention;

public class ReadyForReplayIntention extends ClientIntention {
	public String roomId;

	public ReadyForReplayIntention(String playerName, String roomId) {
		super(ClientIntentionType.ReadyForReplay, playerName);

		this.roomId = roomId;
	}

}
